wallet/encrypter/encrypter.go

```diff
@@ -47,11 +48,20 @@ const (
 	nameParamIterations  = "iterations"
 	nameParamMemory      = "memory"
 	nameParamParallelism = "parallelism"
+	nameParamKeyLen      = "keylen"
 
 	nameFuncNope      = ""
 	nameFuncArgon2ID  = "ARGON2ID"
 	nameFuncAES256CTR = "AES_256_CTR"
+	nameFuncAES256CBC = "AES_256_CBC"
 	nameFuncMACv1     = "MACV1"
+
+	// Parameter Choice
+	// https://www.rfc-editor.org/rfc/rfc9106.html#section-4
+	defaultIterations  = 3
+	defaultMemory      = 65536 // 2 ^ 16
```

Comment:
This method improves readability while keeping the memory definition precise.

```suggestion
	defaultMemory      = 64 << 20 // 2 ^ 16 = 64 MB
```

Reply:
I understand 2 << 16 or (65536) better than 64 << 20!

Reply:
Ok change to `2 << 16` it's better.